cas-server-core-authentication/src/main/java/org/jasig/cas/web/flow/AuthenticationExceptionHandler.java

```diff
@@ -74,17 +79,46 @@
     private String messageBundlePrefix = DEFAULT_MESSAGE_BUNDLE_PREFIX;
 
     /**
-     * Sets the list of errors that this class knows how to handle.
+     * Sets the list of custom exceptions that this class knows how to handle.
+     *
+     * This implementation adds the provided list of exceptions to the default list
+     * or just returns if the provided list is empty.
+     *
+     * This implementation relies on Spring's property source configurer, SpEL, and conversion service
+     * infrastructure facilities to convert and inject the collection from cas.properties.
+     *
+     * This method is thread-safe. It should only be called by the Spring container during application context bootstrap
+     * or unit tests.
      *
      * @param errors List of errors in order of descending precedence.
      */
+    @Value("#{'${cas.custom.authentication.exceptions:}'.split(',')}")
     public void setErrors(final List<Class<? extends Exception>> errors) {
-        this.errors = errors;
+        //The specifics of the default empty value: this results in the list with one null element. So just get rid of null and have
+        //an empty list as a result.
+        final List<Class<? extends Exception>> nonNullErrors = errors.stream().filter(Objects::nonNull).collect(toList());
+        if(nonNullErrors.size() == 0) {
```

Comment:
Checkstyle will likely error out, that this should be `if (nonNullErrors.isEmpty()) {